content/docs/gke/deploy/delete-cli.md

```diff
@@ -40,15 +40,15 @@ For further background about the above settings, see the guide to
 Run the following commands to delete your deployment and reclaim all GCP
 resources:
 
-```
-# If you want to delete all the resources, including storage:
-kfctl delete -f ${CONFIG_FILE} --delete_storage
+  ```
```

Comment:
I don't think this indentation change is necessary. The page looks good on master (https://master.kubeflow.org/docs/gke/deploy/delete-cli/) but in the preview for this PR, there are now unnecessary empty spaces in the code snippet: https://deploy-preview-1645--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/delete-cli/